### ISSUE TITLE
[NEW ICON] Numpy, Scikitlearn, Matplotlib, Power BI, Excel, Seaborn, Pandas

### ISSUE THREAD

**Issue by itscoderhimanshu** [tandpfun/skill-icons#566]
### Skill Name

Numpy, Scikit learn, Pandas

### Why?

These are the important tools used by data scientists and machine learning engineers

## following are the icons
numpy - python library for data science
<img src="https://cdn.worldvectorlogo.com/logos/numpy-1.svg" width="45"/>

pandas - python library for data sciene
<img src="https://cdn.worldvectorlogo.com/logos/panda-7.svg" width="55"/>

matplotlib - python library for charts
<img src="https://seeklogo.com/images/M/matplotlib-logo-7676870AC0-seeklogo.com.png" width="50" alt="matplotlib"/>

scikit learn - python library for machine learning
<img src="https://seeklogo.com/images/S/scikit-learn-logo-8766D07E2E-seeklogo.com.png" width="75"/>

powr bii - analysis tool for data scientists
<img src="https://seeklogo.com/images/P/power-bi-icon-logo-E1B451ED39-seeklogo.com.png" width="30"/>

microsoft excel
<img src="https://seeklogo.com/images/M/microsoft-excel-logo-F8C90B4427-seeklogo.com.png" width="46"/>

seaborn
<img src="https://seeklogo.com/images/S/seaborn-logo-244EB2DEC5-seeklogo.com.png" width="46"/>

**Comment by senolcemhan98** [comment]
@LelouchFR I noticed you already linked this issue to PR. However There is no MS Excel logo yet. It's really important tool for data analytics. I've seen 3 issues about MS Excel logo. Hope you can see this message and add the MS Excel logo asap.

**Comment by LelouchFR** [comment]
> @LelouchFR I noticed you already linked this issue to PR. However There is no MS Excel logo yet. It's really important tool for data analytics. I've seen 3 issues about MS Excel logo. Hope you can see this message and add the MS Excel logo asap.

Yeah I'm aware of that, it was made intentionally, because I would like to add not only one ms thing, but like all the suite and the libre office suite

**Comment by senolcemhan98** [comment]
> > @LelouchFR I noticed you already linked this issue to PR. However There is no MS Excel logo yet. It's really important tool for data analytics. I've seen 3 issues about MS Excel logo. Hope you can see this message and add the MS Excel logo asap.
> 
> Yeah I'm aware of that, it was made intentionally, because I would like to add not only one ms thing, but like all the suite and the libre office suite

Oh amazing! That's really good idea. Thank you for your information, hope to see them asap  🙌 🙌

**Comment by LelouchFR** [comment]
hi @senolcemhan98 I got some news, I made a "fork" of the project (actually just rewritten it in go), so I added these icons in it, so if you want to use them, they are finally released, in rolling release, with some better performances and all of that type of stuff [here](https://github.com/lelouchfr/skill-icons), if you have other ideas of icons, you can open a new issue on my repository ;)

(since the maintainer didn't showed in literally months)
